USER_GUIDE.md

```diff
@@ -1,17 +1,18 @@
 # Document Service Backup Restore Tool
 
 Backup operation reads your documents from the Document service
-using the Document Backup service and writes them as extended JSONs to files in a designated directory. 
+using the Document Backup service and writes them as extended JSONs to files in a designated directory.
 This directory can be later used for restore process.
-  
-Both operations have an option to choose an environment. `dbr` supports all available environments 
-(__us-prod__, __us-stage__, __eu__). With this, it is possible to migrate data from one environment to another.
-     
-Documents are being inserted as they are written in the backup files with all metadata and data types preserved. 
+
+Both operations have an option to choose an environment. `dbr` supports all available environments
+(__us-prod__, __us-stage__, __beta__, __eu__). With this, it is possible to migrate data from one environment to another.
+You will notice that __beta__ and __us-prod__ are exactly the same environment. 
```

Comment:
Please note that both **`__beta__`** and **`__us-prod__`** are exactly the same environments.